_includes/root_path.html

```diff
@@ -0,0 +1,18 @@
+{% assign ROOT_PATH = ''
```

Comment:
This currently produces bad links when `assets/elements/404.md` is rendered as the content of that page is served at the URL of a non-existent page (e.g. https://bitcoincore.org/foo/bar ), and that URL may be more or less deep than the static 404 page.  I think maybe you could throw a `{% if page.permalink == "/404.html" %}{% assign ROOT_PATH="" %}{% endif %}` in there to fix, since I suspect IFPS doesn't have an automatic 404 redirect feature.

Reply:
Added the switch.

Since go-ipfs 0.7.0, the built-in HTTP gateway [supports custom 404 pages](https://github.com/ipfs/go-ipfs/blob/v0.7.0/CHANGELOG.md#custom-404). But I think you wouldn't configure a specific page for a single target/website, but a generic one for the gateway itself instead.

Not sure if this means the 404 links on bitcoincore.org would now be broken when looking at a nested path.